README.md

```diff
@@ -73,6 +73,10 @@ The majority of the questions were plucked from an [oksoclap](http://oksoclap.co
 * Consider HTML5 as an open web platform. What are the building blocks of HTML5?
 * Describe the difference between cookies, sessionStorage and localStorage.
 * Can you explain the difference between `GET` and `POST`?
+* How can you define custom HTML Elements? 
+  * HTML5? Shadow DOM? "Web components"?
+  * Libraries: Polymer? Bosonic? Xtags?
+  * Browser adoption?
```

Comment:
Is this a question? Are you saying that a potential answer is to just wait for native browser adoption of web components? Are you looking for a candidate to talk about the native web component spec?